driver/bpf/fillers.h

```diff
@@ -249,13 +249,20 @@ FILLER(sys_open_x, true)
 	if (res != PPM_SUCCESS)
 		return res;
 
+	bpf_get_fd_dev_ino(retval, &dev, &ino);
```

Comment:
Since we are no more interested in the return value, the function can be `void` instead of returning a `bool`